apps/gitness/src/pages-v2/pull-request/pull-request-compare.tsx

```diff
@@ -288,6 +295,20 @@ export const CreatePullRequest = () => {
       default: false
     }))
   }, [tags])
+  const [selectedCommit, setSelectedCommit] = useState<CommitSelectorListItem>({
```

Comment:
can we create a CommitsStore instead of keeping stuff in state?

Reply:
sure